lib/jobs/ssh-job.js

```diff
@@ -50,7 +46,9 @@ function sshJobFactory(
         return waterline.nodes.needByIdentifier(self.nodeId)
         .then(function(node) {
             return Promise.reduce(self.commands, function(results, commandData) {
-                return self.sshExec(commandData, node.sshSettings, new ssh.Client())
+                return commandUtil.sshExec(
```

Comment:
You need tweak this code and below ones as this PR (https://github.com/RackHD/on-tasks/pull/172) has been merged for hotfix.